docs/feature_pointing_device.md

```diff
@@ -27,6 +27,10 @@ Once you have made the necessary changes to the mouse report, you need to send i
 
 When the mouse report is sent, the x, y, v, and h values are set to 0 (this is done in `pointing_device_send()`, which can be overridden to avoid this behavior).  This way, button states persist, but movement will only occur once.  For further customization, both `pointing_device_init` and `pointing_device_task` can be overridden.
 
+Additionally, by default, `pointing_device_send()` will only send a report when the report has actually changed.  This prevents it from continuously sending mouse reports, which will keep the host system awake.  This behavior can be changed by creating your own `pointing_device_send()` function.
+
+Also, you use the `has_mouse_report_changed(old, new)` function to check to see if the report has changed. 
```

Comment:
Unless I've gotten *really* bad at C, it seems this should be:

```suggestion
Also, you use the `has_mouse_report_changed(new, old)` function to check to see if the report has changed.
```

Reply:
Nah, I'm just bad at documentation